src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const stylesObject = {};
+
+  const lines = sourceString.split(';');
+
+  lines.forEach(function (line) {
+    const part = line.split(':');
+
+    if (part.length >= 2) {
+      stylesObject[part[0].trim()] = part[1].trim();
+    }
+    
+  });
+
+  return stylesObject;
```

Comment:
The function should handle empty lines and lines with only semicolons. Currently, such lines will create an entry with an undefined property name and value.